src/compute-client/src/controller.rs

```diff
@@ -163,6 +163,7 @@ pub enum ComputeControllerResponse<T> {
 pub struct ComputeReplicaConfig {
     pub location: ComputeReplicaLocation,
     pub logging: ComputeReplicaLogging,
+    pub idle_arrangement_merge_effort: Option<u32>,
```

Comment:
Can you please add docs about what this does, since its meaning is much less obvious than that of the other members of this struct